netci.groovy

```diff
@@ -82,23 +55,23 @@ class WcfUtilities
     def wcfPRServiceCount = 0 
     
     // Outerloop jobs for WCF Core require an external server reference
-    // This should be run 
+    // Jenkins provides the correct parameters to the scripts to get the correct PR ID or branch
     def addWcfOuterloopTestServiceSync(def job, String os, String branch, boolean isPR) { 
 
-        // Exclude rc2 branch, since that branch will not have the sync scripts in
-        if (branch.toLowerCase().contains("rc2")) {
-            return 
-        }
-
-        wcfPRServiceCount++
-
         def operation = isPR ? "pr" : "branch"
+        def currentWcfPRService = wcfPRServiceCount++ 
```

Comment:
Aah! I thought I changed that already. But yes, you're right. I will change that prior to merge.